categories/javascript.json

```diff
@@ -31,6 +31,28 @@
       "vanilla"
     ]
   },
+  {
+    "name": "Ditching jQuery",
+    "description": "A growing collection of native JavaScript equivalents for common jQuery tasks.",
+    "url": "https://gomakethings.com/ditching-jquery/",
+    "free": true,
+    "tags": [
+      "jquery",
+      "javascript",
+      "vanilla"
+    ]
+  },
+  {
+    "name": "HtmlDom.dev",
```

Comment:
From a formatting standpoint should HTML and DOM not be capitalised?

Reply:
Updated this to `htmldom.dev` which is the same styling used on the github repository for that project